mne/epochs.py

```diff
@@ -2977,11 +2977,11 @@ def _ensure_list(x):
         *last_cols,
     ]
 
-    data = np.empty((len(events_df), len(columns)))
+    data = np.empty((len(events_df), len(columns)), float)
     metadata = pd.DataFrame(data=data, columns=columns, index=events_df.index)
 
     # Event names
-    metadata.iloc[:, 0] = ""
+    metadata["event_name"] = ""
```

Comment:
... and Pandas does not seem to complain when you set an entire col (or list of cols) rather than using `.iloc`, so I go that route here

Reply:
this one is okay